tools/scope_upload_classification.py

```diff
@@ -177,17 +175,16 @@ def upload_classification(
                     src_dict[src_id] = src
 
                     if match_ids:
-                        warnings.warn(
-                            "One source may have more than one ZTF ID. It is recommended to match to an exising source's obj_id by setting -use_existing_obj_id."
-                        )
+                        print("Attempting to match with survey_id annotation.")
                         annotations = src['annotations']
                         for annot in annotations:
-                            if annot['origin'] == ztf_origin:
-                                src_ztf_ids = [int(x) for x in annot['data'].values()]
-                                if ztfid in src_ztf_ids:
+                            if annot['origin'] == 'SCoPe':
```

Comment:
Don't we want this tunable? Or don't we care?

Reply:
@mcoughlin Probability a good idea to make it tunable; just made the change